flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/config/OptimizerConfigOptions.java

```diff
@@ -362,6 +362,17 @@ public class OptimizerConfigOptions {
                                     + "it receives incremental accumulators and outputs incremental results). "
                                     + "In this way, we can reduce some state overhead and resources. Default is enabled.");
 
+    @Documentation.TableOption(execMode = Documentation.ExecMode.STREAMING)
+    public static final ConfigOption<Integer> TABLE_OPTIMIZER_PTF_MAX_TABLES =
+            key("table.optimizer.ptf.max-tables")
+                    .intType()
+                    .defaultValue(20)
```

Comment:
out of curiosity do we know if there is any other vendor offering PTF feature and what limit do they have?